filters.json

```diff
@@ -222,16 +222,19 @@
 		"description": "Filter the \"People You May Know\" story that appears in the News Feed occasionally."
 	}, {
 		"id": 11,
+		"updated_on": 1493344578822,
```

Comment:
This is a local value in a user's prefs to show when their subscription was last updated. It shouldn't be included in the json file. I will merge and remove it.